docs/about/contributing.md

```diff
@@ -41,148 +41,163 @@ We use [semantic-release](https://www.npmjs.com/package/semantic-release), which
+
+The [Guide][guide-repo] is documentation! Everything you ever hoped to know about the Screwdriver project.
+The [Homepage][homepage-repo] is the basic one-pager that powers [Screwdriver.cd][homepage].
 
-This documentation! Everything you ever hoped to know about the Screwdriver project.
+### [UI][ui-repo]
+
+The Ember-based user interface of Screwdriver.
 
 ### Miscellaneous Tools
 
+* **[circuit-fuses][circuit-fuses-repo]**: Wrapper to provide a node-circuitbreaker with callback interface
 * **[client][client-repo]**: Simple Go-based CLI for accessing the Screwdriver API
 * **[gitversion][gitversion-repo]**: Go-based tool for updating git tags on a repo for a new version number
-* **[circuit-fuses][circuit-fuses-repo]**: Wrapper to provide a node-circuitbreaker w/ callback interface [![Version][circuit-fuses-npm-image]][circuit-fuses-npm-url]
-* **[keymbinatorial][keymbinatorial-repo]**: Generates the unique combinations of key values by taking a single value from each keys array [![Version][keymbinatorial-npm-image]][keymbinatorial-npm-url]
+* **[keymbinatorial][keymbinatorial-repo]**: Generates the unique combinations of key values by taking a single value from each keys array
+* **[toolbox][toolbox-repo]**: Repository for handy Screwdriver-related scripts and other tools
+* **[hashr][hashr-repo]**: Wrapper module for generating ids based on a hash
 
 ### Adding a New Screwdriver Repo
 
 We have some tools to help start out new repos for screwdriver:
 
 * **[generator-screwdriver](https://github.com/screwdriver-cd/generator-screwdriver)**: Yeoman generator that bootstraps new repos for screwdriver
-* **[eslint-config-screwdriver](https://github.com/screwdriver-cd/eslint-config-screwdriver)**: Our eslint rules for node-based code. Included in each new repo as part of the bootstrap process
+* **[eslint-config-screwdriver](https://github.com/screwdriver-cd/eslint-config-screwdriver)**: Our ESLint rules for node-based code. Included in each new repo as part of the bootstrap process
 
-[api-repo]: https://github.com/screwdriver-cd/screwdriver
-[api-npm-image]: https://img.shields.io/npm/v/screwdriver-api.svg
-[api-npm-url]: https://npmjs.org/package/screwdriver-api
-[api-issues-image]: https://img.shields.io/github/issues/screwdriver-cd/screwdriver.svg
-[api-issues-url]: https://github.com/screwdriver-cd/screwdriver/issues
+If you create a new repo, please come back and edit this [page](contributing-docs) so that others can know where your repo fits in.
 
-[launcher-repo]: https://github.com/screwdriver-cd/launcher
+### Screwdriver.cd Tests and Examples
 
-[executor-base-repo]: https://github.com/screwdriver-cd/executor-base
-[executor-base-npm-image]: https://img.shields.io/npm/v/screwdriver-executor-base.svg
-[executor-base-npm-url]: https://npmjs.org/package/screwdriver-executor-base
+The organization [screwdriver-cd-test][screwdriver-cd-test-org] contains various tests and examples for Screwdriver.cd.
```

Comment:
I feel the wording here is a bit confusing. They are not the tests but repositories used in acceptance tests.